npc/custom/woe_controller.txt

```diff
@@ -175,13 +175,17 @@ OnReward:
 					setd ".@ip_"+.@i+"_"+.@ip$,1;
 				}
 				if (.Options&2) {
-					query_sql("INSERT INTO `mail` (send_name,dest_id,title,message,nameid,amount,identify,zeny,time) VALUES ("+
-					          "'no-reply',"+.@cid[.@j]+",'** Siege Reward: "+getcastlename(.Castles$[.@i])+" **',"+
-					          "'Brave one,% % Congratulations!% Your guild has successfully occupied% territory in the War of Emperium on% "+.@str$+".% % % % % [ Your reward is attached. ]',"+
-					          .Reward[0]+","+.Reward[1]+",0,"+.Reward[2]+",UNIX_TIMESTAMP(NOW()))");
+					.@charid = .@cid[.@j];
+					.@sender$ = "no-reply";
+					.@title$ = "** Siege Reward: "+getcastlename(.Castles$[.@i])+" **";
+					.@body$ = "Brave one,\r\n \r\n Congratulations!\r\n Your guild has successfully occupied\r\n territory in the War of Emperium on\r\n "+.@str$+".\r\n \r\n \r\n \r\n \r\n [ Your reward is attached. ]";
+					setarray .@mailitem[0], .Reward[0];
+					setarray .@mailamount[0], .Reward[1];
+					mail .@charid, .@sender$, .@title$, .@body$, .Reward[2], .@mailitem, .@mailamount;
+
 					if (!getd(".@str_"+.@cid[.@j]) && isloggedin(.@aid[.@j],.@cid[.@j])) {
```

Comment:
If we are already touching this, maybe put a packetver condition into this as well, because you will get a duplicate notification with this on rodex. Because rodex already notifies you.